github/repos.go

```diff
@@ -538,6 +540,12 @@ type RequiredPullRequestReviews struct {
 	IncludeAdmins bool `json:"include_admins"`
```

Comment:
@shurcooL I'm not sure about deprecation, seems like an overhead, maybe it's better just to remove old fields.